src/main/groovy/de/dkfz/roddy/execution/jobs/cluster/lsf/rest/LSFRestJobManager.groovy

```diff
@@ -334,8 +325,9 @@ class LSFRestJobManager extends BatchEuphoriaJobManagerAdapter {
      * Updates job information for given jobs
      * @param jobList
      */
-    void getJobDetails(List<BEJob> jobList) {
+    private Map<BEJobID,GenericJobInfo> getJobDetails(List<BEJobID> jobList) {
```

Comment:
If you enable background updating in BEJobManager, the cache will be updated regularly, you can even set the interval.

Reply:
Right, but as it is in the moment, the method has the side effect to update the cache (always). Maybe the user only wants the current cache value and otherwise simply rely on the background updater? That is at least what I'd do, if the updating process is slow (network).

Reply:
I renamed it and also the getJobStates methods